contrib/dokku/Dockerfile

```
@@ -0,0 +1 @@
+FROM codeforafrica/twoopstracker_ui:0.1.0-alpha.0
```

```suggestion
FROM codeforafrica/twoopstracker-ui:0.1.0-alpha.0
```

Not a strict requirement but I think most images use `-` instead of `_` in names.